src/Controls/samples/Controls.Sample/Maui.Controls.Sample-net6.csproj

```diff
@@ -38,6 +41,15 @@
     <MauiFont Include="Resources\Fonts\*" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Content Remove="wwwroot\css\app.css" />
+    <Content Remove="wwwroot\index.html" />
+  </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="wwwroot\css\app.css" />
```

Comment:
I'm wondering if we may eventually want to switch these up to use `MauiAsset` when we add support for that.  The idea would be then that any items are added to the appropriate platform specific item group to be included as assets (ie: Assets folder on android, Resources on iOS, etc).  Might be a bit faster to load these from the asset locations instead of having to get them out of the managed assembly at runtime?

Reply:
That could be interesting for sure. All we need in Blazor is _some_ way to get to these static assets. On Windows typically it's using a file system provider, because you can do that on Windows. For now we use embedded resources because it works everywhere.